clubs/models.py

```diff
@@ -279,6 +283,20 @@ def __str__(self):
         return self.name
 
 
+class Badge(models.Model):
+    label = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+
+    # The color of the badge to be displayed on the frontend.
+    color = models.CharField(max_length=16, default='')
```

Comment:
Should we use a custom validator here to ensure the field is an html color code (or the like)?

Reply:
We could; I don't think it's an issue since browsers will just ignore a color that doesn't make sense, it won't throw an error or anything.